it/v1/endpoints/RetrieveChargeHistoryControllerISpec.scala

```diff
@@ -110,7 +110,7 @@ class RetrieveChargeHistoryControllerISpec extends IntegrationBaseSpec {
 
         val response: WSResponse = await(request.get)
         response.status shouldBe INTERNAL_SERVER_ERROR
-        response.json shouldBe Json.toJson(DownstreamError)
+        response.json shouldBe Json.toJson(InternalError)
```

Comment:
I think all the ISpecs could do with this simplification: 
```suggestion
        response.json shouldBe InternalError.asJson
```